api/v1alpha1/backend_types.go

```diff
@@ -131,7 +131,6 @@ type BackendSpec struct {
 	// the health of the active backends falls below 72%.
 	//
 	// +optional
-	// +notImplementedHide
```

Comment:
Nit: a link to https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/upstream/load_balancing/priority#arch-overview-load-balancing-priority-levels will help users to understand how fallback works.

Reply:
we dont want to expose the internal details, only want to represent it as a boolean field